SwiftUIDZ/SwiftUIDZ/ProfileRowView.swift

```diff
@@ -0,0 +1,64 @@
+// ProfileRowView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Ячейка для экрана профиля
+struct ProfileRowView: View {
+    // MARK: Constants
+
+    private enum Constants {
+        static let yellowColorName = "pastelYellow"
+        static let orangeColorName = "pastelOrange"
```

Comment:
```suggestion
        static let yellowColorName = "pastelYellow"
        static let orangeColorName = "pastelOrange"
```
а что не вынесла в расширение цвета?